### DISCUSSION TITLE
venv location appended permanently to $PATH

### DISCUSSION THREAD

**Discussion by slikas** [coder/code-server#4863] [Q&A]
Just yesterday I tried to run Calibre which kept reporting missing module: msgpack. By running "which python" in a regular bash, apparently the python version wrongly used (~/project/.venv/bin/python) is the one in the virtual environment for another project I've been coding inside code-server. 

(sudo) echo $PATH prints: ~/project/.venv/bin:/usr/lib/code-server/vendor/modules/code-oss-dev/bin:... Are they supposed to be there universally outside of venv? There was no such error before yesterday. If not how do I remove them? 

Running the regular albeit verbose one "/usr/bin/python /usr/bin/calibre" works fine

**Comment by lg-nineoneninethreenine** [comment]
Hmm, all I can think of is that the VENV made a symlink to the folder where Linux commands are stored. Try removing the old symlink, or maybe an `alias` in ~/.bashrc, but make sure to create a dotfiles repo with an install script like @jsjoeio's that replaces the old bash rc file with your file. It seems like you use Docker, hence try setting up rsync or make a dotfiles repo with a Linux shell script. Other than that, 🤷
